### ISSUE TITLE
[v2] move the quotes to remark admonitions

### ISSUE THREAD

**Issue by yangshun** [facebook/docusaurus#2304]
## 📚 Documentation

We have a bunch of blockquotes in our documentation which looks like [this](https://v2.docusaurus.io/docs/cli#docusaurus-swizzle) and they are not very obvious because of the changes in v2's CSS. 

Ever since callouts/admonitions have been [added in 2.0.0.alpha.43](https://v2.docusaurus.io/docs/markdown-features#calloutsadmonitions) by the awesome @elviswolcott , we can use them in place of the blockquotes.

The task here is to go through the docs and help replace the use of blockquotes with the suitable admonitions. Refer to the [admonitions docs](https://github.com/elviswolcott/remark-admonitions) for the available types.

Note that you do not have to convert *all* the blockquotes in the docs, you are free to just do whichever you please, but please comment here so that other interested contributors are aware and the contributions don't conflict.

**Tip:** The docs for the master version are found under https://localhost:3000/docs/next not https://localhost:3000/docs/ (that is the current released version)

### Have you read the [Contributing Guidelines on issues](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#reporting-new-issues)?

Yes

**Comment by Zhencha0Cai** [comment]
Hi @yangshun , I have finished work to update quote to  remark admonitions. However, I find that the pre-commit prettier task mess up the the format of code. It removed the line break before and after the `:::`.
Any pointer to fix this? 
https://github.com/Zhencha0Cai/docusaurus/commit/71c91d4928cd7048e4fb3a25f62182b3ce12311c

**Comment by yangshun** [comment]
Hmm I think you need to add a title. Try the following formats:

```
:::tip Title
The content and title *can* include markdown.
:::
```

or add an empty line after the `:::`

```
:::tip

The content and title *can* include markdown.
:::
```

**Comment by Zhencha0Cai** [comment]
@yangshun. Thanks for the help.  I have to use this format to make it work.
```
:::tip

The content and title *can* include markdown.

:::
```

Here is the PR. https://github.com/facebook/docusaurus/pull/2313

**Comment by lex111** [comment]
Resolved in #2313
